addons/common/functions/fnc_uniqueItems.sqf

```diff
@@ -0,0 +1,38 @@
+/*
+ * Author: mharis001
+ * Returns list of unique items in a unit's inventory.
+ * Items are cached if unit is ACE_player.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Items <ARRAY>
+ *
+ * Example:
+ * [_player] call ace_common_fnc_uniqueItems
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+#define ITEMS_CACHE_TIME 3600
+
+params ["_unit"];
+
+private _fnc_getItems = {
+    params ["_unit"];
```

Comment:
That is just the CBA thing without assignedItems right? https://github.com/CBATeam/CBA_A3/pull/922 add's a parameter to disable assignedItems. But this specific implementation in here will be faster. Than the CBA one with it's conditionals.

Btw you don't need `params ["_unit"]` the local variable implicitly carries over.

Reply:
And you are right about `_unit`, so I guess `cachedCall` doesn't need anything passed to it either.

Reply:
And you are right about `_unit`, i guess just pass `0` to `cachedCall`